src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -133,25 +135,28 @@ ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
         // Transferring from MatMul representation: [B, I, K] * [B, K, O] = [B, I, O]
         // to FullyConnected representation: [I, K] * [K, O] = [I, O]
 
-        // Weights normalization
-        if (!matmul->get_transpose_b()) {
-            fc_input_b = create_transpose(fc_input_b, matmul->get_friendly_name() + "/transpose_b");
-        }
-
         if (rank_b != 2) {
             ngraph::Dimension K = *(shape_b_aligned.rbegin() + 1);
             NGRAPH_CHECK(K.is_static());
-            std::vector<int64_t> reshape_shape_values = { -1ll, static_cast<int64_t>(K.get_length()) };
-            auto reshape_shape = ngraph::opset1::Constant::create(ngraph::element::i64, ngraph::Shape{ 2 }, reshape_shape_values);
+            std::vector<int64_t> reshape_shape_values = {K.get_length()};
```

Comment:
minor: 
maybe it's better from readability perspective
std::vector<int64_t> reshape_shape_values(2);
auto k_len =  K.get_length();
matmul->get_transpose_b() ? reshape_shape_values = { -1, k_len} : reshape_shape_values  = {k_len, -1};

Reply:
I agree, your version is more readable. But I changed it a little bit: hope the readability is not affected by this.